lib/message.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/database_connection'
+
+class Message
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    sql_query = "SELECT * FROM peeps"
+
+    result = DatabaseConnection.query(sql_query)
+    result.map do |message|
+      Message.new(id: message['id'], message: message['message'], date: message['created_date'])
```

Comment:
Good thinking/ planning ahead here. Amazing to include the date aspect so early.